src/Sulu/Bundle/AdminBundle/Resources/js/components/Tabs/Tabs.js

```diff
@@ -197,6 +192,10 @@ class Tabs extends React.Component<Props> {
         let visibleTabIndices: number[] = [];
 
         const childIndices = this.childIndices;
+
+        // Sorts childIndices in it's natural order, except that the element with selectedIndex is positioned at the
+        // first place and the element withlastShownIndex is positioned at the second place.
```

Comment:
`with lastShownIndex`, but wait with fixing until we have decided the previous comment.